tests/unit/src/test/scala/tests/CallHierarchyLspSuite.scala

```diff
@@ -0,0 +1,71 @@
+package tests
+
+class CallHierarchyLspSuite extends BaseCallHierarchySuite("call-hierarchy") {
+  test("def-incoming-call") {
+    for {
+      result <- assertPrepareIncomingCalls(
+        """|/a/src/main/scala/a/Main.scala
+           |package a
+           |
+           |object Main {
+           |  def @@a() {}
+           |  def <<b>>/*1*/() { <?<a>?>/*1*/() }
+           |  def c() { b() }
+           |  def <<d>>/*2*/() { <?<a>?>/*2*/() }
+           |}
+           |
+           |""".stripMargin
+      )
+      _ <- assertIncomingCalls(
+        """|/a/src/main/scala/a/Main.scala
+           |package a
+           |
+           |object Main {
+           |  def a() {}
+           |  def b() { a() }
+           |  def <<c>>/*1*/() { <?<b>?>/*1*/() }
+           |  def d() { a() }
+           |}
+           |
+           |""".stripMargin,
+        result("1"),
+      )
+    } yield ()
+  }
+
+  test("def-outgoing-call") {
+    for {
+      result <- assertPrepareOutgoingCalls(
```

Comment:
Looks good! Though I think we can just used `assertOutgoingCalls` in both places. The same in `assertIncomingCalls`